backend/users-service/app/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+export enum Role {
```

Comment:
just fyi - the API Gateway's defined role types are here
`backend/api-gateway/app/types/user.ts`